galleries/users_explain/index.rst

```diff
@@ -1,10 +1,40 @@
 .. _users-guide-explain:
 .. _users-guide-using:
+.. _users-guide-index:
+
 
 .. redirect-from:: /users/explain
+.. redirect-from:: /contents
 
 Using Matplotlib
-----------------
+================
+
+
+.. grid:: 1 1 2 2
+
+   .. grid-item::
+
+      .. image:: ../../_static/anatomy.png
```

Comment:
I still object to this, as in https://github.com/matplotlib/matplotlib/issues/26205 as necessary use of viewer real estate.